.vscode/tasks.json

```diff
@@ -0,0 +1,86 @@
+{
+	"tasks": [
```

Comment:
In general I would prefer not to include IDE specific files in a repo. Especially since that means that whoever maintains that repo will then have to maintain all the config files for those IDEs, even if they don't use it themselves. Making these tasks into "convenience" npm commands, and documenting them would be a better alternative.